src/facade/dragonfly_listener.cc

```diff
@@ -56,23 +56,14 @@ using absl::GetFlag;
 namespace {
 
 #ifdef DFLY_USE_SSL
-// To connect: openssl s_client  -cipher "ADH:@SECLEVEL=0" -state -crlf  -connect 127.0.0.1:6380
-static SSL_CTX* CreateSslCntx() {
+SSL_CTX* CreateServerSslCntx() {
```

Comment:
can you please adjust the explanation on how to connect to dragonfly with openssl cli client?